server-phoenix/lib/helios/slack_channel_names.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Helios.SlackChannelNames do
+  use Ecto.Schema
+
+  import Ecto.Changeset
+  import Ecto.Query
+
+  schema "slack_channel_names" do
+    # make these snake case
+    field :channel_name, :string
+    field :channel_id, :string
+
+    timestamps()
+  end
+
+  def with_id(query, channel_id) do
+    from(q in query, where: q.channel_id == ^channel_id, select: {q.channel_id})
+  end
+
+  def name_to_id(query, channel_id) do
+    from(q in query, where: q.channel_id == ^channel_id, select: q.channel_name)
+  end
+
+  # if no slack id and if slack id exists
```

Comment:
Go ahead and take out commented code.  If it belongs to another commit, just stage the lines you need instead of commenting.